src/main/java/io/codelex/java_advanced_test/Exercise2/Basket.java

```diff
@@ -0,0 +1,34 @@
+package io.codelex.java_advanced_test.Exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+
+    private List<T> basketItems = new ArrayList<>();
+
+
+    public void addToBasket(T item) {
+        try {
+            basketItems.add(item);
+            if (basketItems.size() == 10) {
+                throw new BasketFullException("Basket Full");
```

Comment:
Šeit tieši tas pats - nav jēgas mest kļūdu un viņu uzreiz noķert. Faktiski šī metode pēc būtības ja grozs ir pilns izvada Basket Full, bet kļūda tiek uzreiz noķerta un tas, kurš to metodi sauca kļūdu nedabūs.

Reply:
Salaboju , bet nesaprotu kāpec tests nostrādā uz 12item nevis 10 :/